src/main/java/org/akip/service/mapper/TaskDefinitionMapper.java

```diff
@@ -0,0 +1,49 @@
+package org.akip.service.mapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.akip.domain.ProcessDefinition;
+import org.akip.domain.TaskDefinition;
+import org.akip.service.dto.ProcessDefinitionDTO;
+import org.akip.service.dto.TaskDefinitionDTO;
+
+import org.mapstruct.BeanMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+import java.util.Collections;
+import java.util.Map;
+
+@Mapper(componentModel = "spring", uses = {})
+public interface TaskDefinitionMapper extends EntityMapper<TaskDefinitionDTO, TaskDefinition> {
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    @Named("name")
+    @BeanMapping(ignoreByDefault = true)
+    @Mapping(target = "id", source = "id")
+    @Mapping(target = "name", source = "name")
+    @Mapping(target = "documentation", source = "documentation")
+    @Mapping(target = "taskId", source = "taskId")
+    @Mapping(target = "formBuilder", source = "formBuilder")
+    @Mapping(target = "formVersion", source = "formVersion")
+    @Mapping(target = "formSchema", source = "formSchema")
```

Comment:
Do you need all this information in this method?

Reply:
I removed the fields I wasn't using